lib/colorize.js

```diff
@@ -2,28 +2,29 @@
 
 function colorize(...args) {
   const str = args.join(' ');
+  const isTTY = Boolean(process.stdout.isTTY);
 
   return {
-    reset: `\x1b[0m${str}\x1b[0m`,
-    dim: `\x1b[2m${str}\x1b[22m`,
+    dim: `${isTTY ? '\x1b[2m' : ''}${str}${isTTY ? '\x1b[22m' : ''}`,
```

Comment:
I think you can create fn like this:

```js
function buildColorize(str, startTag, endTag) {
  return `${isTTY ? startTag : ''}${str}${isTTY ? endTag : ''}`;
}
```

And then use it everywhere:

```js
return {
  dim: buildColorize(str, '\x1b[2m', '\x1b[22m'),
  // ...
}
```

Even though, if you define the fn inside `colorize` you may omit passing `str` and get it from the closure.